docs/pages/docs/reference/telemetry.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Privacy Policy (Telemetry)"
```

Comment:
Why do we want to title it as **Privacy Policy** instead of just **Telemetry**?

Reply:
Because it is also our Privacy Policy